Addressbook/src/module/delete-user.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/ash
+
+. module/search-user.sh
+
+db="../database/addressbook.db"
+
+usage_delete() 
+{ 
+	echo "---------------------------------------"
+	echo "Delete User"
+	echo "	Usage: [-u <name> ] [-e <email>] [-p <phone>]"
+	echo "---------------------------------------" 
+}
+
+usage_delete_user()
+{
+	echo "---------------------------------------"
+	echo "You can't delete|update multiple user at same time or User not found. Try to add different or more complex search criteria"
+	usage_delete
+	echo "---------------------------------------"
+}
+
+confirm()
+{
+	while true; 
+	do
+	    read -p "Do you want to delete|update this user?" yn
+	    case $yn in
+	        [Yy]* ) 
+				coo="sed -i -e \"/${delete}/d\" $db"
+				eval $coo
```

Comment:
Yep, as discussed, `eval` is a no-no, even in shell. (Though not as horrible as in python, for example.)